service/test/lib/integration/teamOpsTest.js

```diff
@@ -154,14 +154,52 @@ lab.experiment('TeamOps', () => {
     })
   })
 
+  // TODO: Needs review
```

Comment:
Or maybe you mean the pagination information should not be inside `teamOps.readTeam`?